compose.yml

```diff
@@ -15,10 +17,36 @@ services:
       context: .
       dockerfile: .devcontainer/Dockerfile
     image: cdt/disaster-recovery:dev
+    depends_on:
+      - postgres
+      - pgweb
     env_file: .env
     # https://code.visualstudio.com/docs/remote/create-dev-container#_use-docker-compose
     entrypoint: sleep infinity
     ports:
       - "${DJANGO_LOCAL_PORT:-8000}:8000"
     volumes:
       - ./:/cdt/app
+
+  postgres:
+    image: postgres:17
+    environment:
+      - POSTGRES_DB
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
+    ports:
+      - "${POSTGRES_PORT:-5432}:5432"
+
```

Comment:
Need the volume mapping here to preserve the database between restarts.

Reply:
https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/commit/6fb24b716caef58292154ccf2ce9fbb67195f960